apps/calendar/build/calendar.build.js

```diff
@@ -17,15 +18,34 @@
 
   modules: [
     {
-      name: 'ext/alameda',
+      create: true,
+      name: 'bundle',
       include: [
+        'ext/curl',
         'main',
-        'app',
-        'debug',
-        'next_tick'
+        'views/errors',
+        'views/month',
+        'views/month_day_agenda',
+        'views/time_header'
       ]
     },
     {
+      create: true,
```

Comment:
yes there was. easier to remove the shared dependencies and since we call `require(['lazy_loaded'])` as soon as `app-loaded` it doesn't make any perceived difference to the user (don't wait any click to happen before triggering the load)